src/plugins/condition/components/Condition.vue

```diff
@@ -21,160 +21,167 @@
-                   type="text"
-                   @blur="persist"
-            >
-        </span>
-
-        <span class="c-cdef__label">Output</span>
-        <span class="c-cdef__controls">
-            <span class="c-cdef__control">
-                <select v-model="selectedOutputSelection"
-                        @change="setOutputValue"
-                >
-                    <option v-for="option in outputOptions"
-                            :key="option"
-                            :value="option"
-                    >
-                        {{ initCap(option) }}
-                    </option>
-                </select>
-            </span>
-            <span class="c-cdef__control">
-                <input v-if="selectedOutputSelection === outputOptions[2]"
-                       v-model="condition.configuration.output"
-                       class="t-condition-name-input"
+        <div v-if="expanded"
+             class="c-condition__definition c-cdef"
+        >
+            <span class="c-cdef__separator c-row-separator"></span>
+            <span class="c-cdef__label">Condition Name</span>
+            <span class="c-cdef__controls">
+                <input v-model="condition.configuration.name"
+                       class="t-condition-input__name"
                        type="text"
                        @blur="persist"
                 >
             </span>
-        </span>
 
-        <div v-if="!condition.isDefault"
-             class="c-cdef__match-and-criteria"
-        >
-            <span class="c-cdef__separator c-row-separator"></span>
-            <span class="c-cdef__label">Match</span>
+            <span class="c-cdef__label">Output</span>
             <span class="c-cdef__controls">
-                <select v-model="condition.configuration.trigger"
-                        @change="persist"
-                >
-                    <option v-for="option in triggers"
-                            :key="option.value"
-                            :value="option.value"
-                    > {{ option.label }}</option>
-                </select>
+                <span class="c-cdef__control">
+                    <select v-model="selectedOutputSelection"
+                            @change="setOutputValue"
+                    >
+                        <option v-for="option in outputOptions"
+                                :key="option"
+                                :value="option"
+                        >
+                            {{ initCap(option) }}
+                        </option>
+                    </select>
+                </span>
+                <span class="c-cdef__control">
+                    <input v-if="selectedOutputSelection === outputOptions[2]"
+                           v-model="condition.configuration.output"
+                           class="t-condition-name-input"
+                           type="text"
+                           @blur="persist"
```

Comment:
lets use @change here instead. That will prevent unnecessary persist calls

Reply:
I made the change, but in testing it I found that it made no difference for number of calls to persist. Probably due to the v-bind there is no difference?

Reply:
understand this fully now